recipe/meta.yaml

```diff
@@ -17,18 +17,21 @@ requirements:
     - setuptools
     - cython >=0.23
     - python
-    - six >=1.4
+    - six >=1.7.3
     - numpy x.x
-    - scipy >=0.9
+    - scipy >=0.17
+    - numpydoc >=0.6
   run:
     - python
-    - six >=1.4
-    - numpy x.x
```

Comment:
Ah. For some reason in my brain it was only needed in the build! Sorry for that, my bad. I'll re-add and (maybe?) re-render.